int8 controller_to_execute  # controller enum, defined in egmri_pb2

TfParams tf
